tests/unit_tests/blas/include/reference_blas_templates.hpp

```diff
@@ -1127,12 +1190,12 @@ void rotg(double *a, double *b, double *c, double *s) {
 
 template <>
 void rotg(std::complex<float> *a, std::complex<float> *b, float *c, std::complex<float> *s) {
-    crotg_((void *)a, (void *)b, c, (void *)s);
+    onemkl_crotg((void *)a, (void *)b, c, (void *)s);
 }
 
 template <>
 void rotg(std::complex<double> *a, std::complex<double> *b, double *c, std::complex<double> *s) {
-    zrotg_((void *)a, (void *)b, c, (void *)s);
+    onemkl_zrotg((void *)a, (void *)b, c, (void *)s);
```

Comment:
I have reservations about this function name. It looks like a onemkl function. Maybe we can call it *_wrapper?

Reply:
renamed to zrotg_wrapper